shared/extension/behaviour/ProviderScope.hpp

```diff
@@ -59,6 +60,21 @@ namespace extension::behaviour {
                 return a.optional == b.optional ? a.priority > b.priority : b.optional;
             });
 
+            // If there are any Continue tasks, do nothing else
+            // If there are other tasks with Continue, they will be ignored
+            for (const auto& task : tasks) {
+                if (task.type == typeid(::extension::behaviour::Continue)) {
+                    // Tell the user if there is a Continue task with other tasks
+                    if (tasks.size() > 1) {
+                        NUClear::log<NUClear::LogLevel::WARN>(
+                            "Continue task was emitted with other tasks, the other tasks will be ignored");
+                    }
+
+                    // Do nothing else
+                    return;
```

Comment:
You need to also clear the tasks list so that next time there isn't still a continue in there (also maybe add a unit test to make sure that doesn't happen)